launcher/src/backend/ethereum-services/PrysmBeaconService.int.js

```diff
@@ -151,7 +153,7 @@ test('prysm validator import', async () => {
     //expect(BCstatus.stderr).not.toMatch(/Could not connect to execution endpoint/)
 
     //check prysm VC logs
-    expect(VCstatus.stderr).toMatch(/Validating for public key/)
     expect(VCstatus.stderr).toMatch(/Beacon chain started/)
+    expect(runningValidator).toMatch(/Showing 3 validator accounts/)
```

Comment:
`3` is not fixed number of validators

Reply:
that's true but i'm only importing 3 so if it's not 3 something did not work

Reply:
okey I see. then should be fine for this case